### ISSUE TITLE
fix(admin-tool): Improve layout of donation export columns

### ISSUE THREAD

**Issue by Sidsector9** [WordImpress/Give#3266]
## User Story
As a user, I want the UI/UX of the donation export columns improved so that on small screens the layout doesn't break.

## Current Behavior
Currently on mid-size screens, the 4th column breaks as shown in the screenshot.

## Expected Behavior
I expect the columns to display properly on mid-size screens.

## Visuals
![breakcolumns](https://snag.gy/VabwHP.jpg)

## Tasks
- [ ] Fix CSS for columns

**Comment by DevinWalker** [comment]
I agree, let's also make sure the text respects the indents as well:

![68747470733a2f2f736e61672e67792f5661627748502e6a7067](https://user-images.githubusercontent.com/1571635/40554256-fe240ae4-5ff9-11e8-9aad-7411aab1b786.png)

**Comment by Sidsector9** [comment]
@DevinWalker This is the revised look

![textindent](https://snag.gy/GNkM95.jpg)
